recipes/datapane/meta.yaml

```diff
@@ -0,0 +1,89 @@
+{% set name = "datapane" %}
+{% set version = "0.6.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 398fdf46259e6e7485e01ddb817fe27158797120346464fbe476c80ac13f8e5b
+  
+build:
+  noarch: python
+  number: 0
+  skip: True  # [py<36]
+  entry_points:
+    - datapane = datapane.client.__main__:main
+    - dp-runner = datapane.runner.__main__:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+    - poetry
+  run:
+    - python
+    - dataclasses  # [py==36]
+    - altair >=4.0.0,<5.0.0
+    - bokeh >=2.0.0,<3.0.0
+    - click >=7.0.0,<8.0.0
+    - click-spinner >=0.1.8,<0.2.0
+    - colorlog >=4.1.0,<5.0.0
+    - dacite >=1.5.0,<2.0.0
+    - flit-core >=2.3.0,<2.4.0
+    - folium >=0.11.0,<0.12.0
+    - importlib_resources >=3.0.0,<4.0.0
+    - jinja2 >=2.11.1,<3.0.0
+    - jsonschema >=3.2.0,<4.0.0
+    - matplotlib >=3.1.0,<4.0.0
+    - munch >=2.5.0,<3.0.0
+    - nbconvert >=5.6.1,<6.0.0
+    - numpy >=1.18.0,<2.0.0
+    - packaging >=20.3,<21.0
+    - pandas >=1.0.1,<2.0.0
+    - plotly >=4.8.1,<5.0.0
+    - pyarrow >=0.17.0,<0.18.0
+    - pyyaml >=5.3.0,<6.0.0
+    - requests >=2.20.0,<3.0.0
+    - ruamel.yaml >=0.16.5,<0.17.0
+    - stringcase >=1.2.0,<2.0.0
+    - tabulate >=0.8.7,<0.9.0
+    - toolz >=0.10.0,<0.11.0
+    - validators >=0.15.0,<0.16.0
+
+
+test:
+  imports:
+    - datapane
+    - datapane.client
+    - datapane.client.api
+    - datapane.client.scripts
+    - datapane.common
+    - datapane.resources
+    - datapane.resources.local_report
+    - datapane.resources.scaffold
+    - datapane.runner
+  commands:
+    - datapane --help
+    - datapane --version
+    - dp-runner --help
+
+about:
+  home: https://datapane.com
+  license: Proprietary
+  license_family: PROPRIETARY
+  license_file: {{ name }-{{ version }}/LICENSE.txt
```

Comment:
```suggestion
  license_file: LICENSE.txt
```

Reply:
This LICENSE file is bundled with the `sdist` in the location indicated by the template above - will conda-build find it in the nested subdirectory?